### ISSUE TITLE
Cannot enabled password as sync item

### ISSUE THREAD

**Issue by A-t-s-u-s-h-i** [brave/brave-browser#40552]
### Description

A user reports that they cannot enable password sync even if biometric auth was passed. It is reproducible on my device(Android13/Brave1.68.137).

<img width="742" alt="スクリーンショット 2024-08-20 11 05 12" src="https://github.com/user-attachments/assets/f2a5be5b-a62f-4cd7-98a1-1ba6f5495527">

https://github.com/user-attachments/assets/c1456eb4-c58e-48b4-a58d-f23df5521f17

### Steps to reproduce

1. Setting => Sync => Select sync option
2. Enable Password
3. Done biometric auth

### Actual result

Password won't be enabled as sync item

### Expected result

It should be enabled

### Reproduces how often

Easily reproduced

### Brave version

1.68.137

### Device

- Brand/model: moto g32
- Android version: Android13

### Channel information

- [X] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [X] with Brave Shields disabled
- [X] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by AlexeyBarabash** [comment]
I was able reproduce with my old Android 9 device where there is no any biometric  sensor, so the graphic pattern is used.

On devices with a fingerprint sensor enable I wasn't able to reproduce it.

**Comment by AlexeyBarabash** [comment]
I wasn't able to reproduce it on my dev build, so I have checked all versions:

Stable `1.69.153` - can reproduce
Beta `1.70.92` - can't reproduce
Nightly `1.71.37` - can't reproduce

**Comment by francoism90** [comment]
dup: https://github.com/brave/brave-browser/issues/36190

I have the same issue on Android 15 Beta (Pixel) and Brave stable.
Same for on a Chromebook.

**Comment by mbasaglia** [comment]
Happens for me as well
Brave 1.70.126, Chromium 129.0.6668.100
Samsung Galaxy A54 5G
Android 14

**Comment by AlexeyBarabash** [comment]
Yes, this is a duplicate of https://github.com/brave/brave-browser/issues/36190 .

Closing as it is fixed with https://github.com/brave/brave-core/pull/26876
